README.md

```diff
@@ -736,8 +739,9 @@ Move the item and lock the position of any element that's located at the same
 position.
 
 - item (`gaphas.item.Item`): The item to move.
-- view (`gaphas.view.View`): The view with guides to to use for move
-coordinates.
+- view (`gaphas.view.View`): The view with guides to use for move
+  
```

Comment:
Why was an extra return added here and on line 762?

Reply:
Done